src/components/Video/Video.styles.css

```diff
@@ -0,0 +1,11 @@
+.video {
+  border-radius: 5px;
+  background-color: #fffdee54;
+  width: 250px;
+  min-height: 300px;
```

Comment:
You can leave it like that. But In the future, it's a good practice to use relative units and save pixels only for font sizes. I'll leave you a link with info about it: 

https://medium.com/@julienetienne/pixels-are-dead-faa87cd8c8b9
https://thecssworkshop.com/lessons/relative-units